client/express-checkout/index.js

```diff
@@ -263,6 +265,7 @@ jQuery( ( $ ) => {
 				};
 				wcpayECE.block();
 				event.resolve( clickOptions );
+				onClickHandler( event );
```

Comment:
> [!NOTE]
> This is a nitpick.

I think I'd want the `.resolve()` call to happen last, just for semantics. To me it feels a little cleaner to have the resolve/return be the last thing that happens in the handler 🤔 

What do you think?

Reply:
I agree 👍 . done in https://github.com/Automattic/woocommerce-payments/pull/8993/commits/38c99d367b081bae113011204b1ab185bfb045d8